src/services/identity/SitesService.ts

```diff
@@ -331,6 +331,21 @@ class SitesService {
       })
     )
 
+    // helper type guard
+    // note: empty strings will return false as well
+    const isString = (val: string | undefined): val is string => !!val
```

Comment:
Style: could we just use the lodash `isString` for this?

Reply:
oh yea! great suggestion haha